modules/graphics/fonts.nix

```diff
@@ -6,12 +6,19 @@
   config,
   ...
 }: let
-  cfg = config.ghaf.graphics.weston;
+  inherit (config.ghaf.graphics) weston;
+  inherit (config.ghaf.graphics) labwc;
```

Comment:
You can combine these to oneline - see example https://github.com/tiiuae/ghaf/blob/95776688fee50323c121ffd7e44f7fb1d976a355/lib/ghaf-modules.nix#L6